SabaCenter/ViewModel/ViewModels/LectureViewModel.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  LectureViewModel.swift
+//  SabaCenter
+//
+//  Created by Mohsen Hosseini on 4/7/18.
+//  Copyright © 2018 Saba. All rights reserved.
+//
+
+import Foundation
+
+public class LectureViewModel {
+    var lecture: LectureItem
+
+    var title: String {
+            return lecture.title
+    }
+
+    var date: String {
+            let dateFormatter = DateFormatter()
+            dateFormatter.locale = Locale(identifier: "en_US")
```

Comment:
Is this supposed to be `"en_US"` or `"en-US"`?

Reply:
It's supposed to be "en_US"
https://developer.apple.com/documentation/foundation/dateformatter